### ISSUE TITLE
Assertions on result of wrapper.get

### ISSUE THREAD

**Issue by marcodejongh** [producthunt/chai-enzyme#12]
Assertions of result of get are currently not possible

```
expect(wrapper.children().get(1)).to.have.tagName('td');
```

Should work the same as

```
expect(wrapper.children().at(1)).to.have.tagName('td');
```

**Comment by ljharb** [comment]
`.get` returns a raw node. How would chai-enzyme differentiate between a raw node, an enzyme wrapper, and any other kind of object?

**Comment by marcodejongh** [comment]
@ljharb Based on the structure you can determine that it is a raw node but we might just want to have it thrown an error if you pass something other then a wrapper to the expect. 

The only reason I need ```.get``` instead of ```.at``` is because the ```tagName``` appears to be broken #13

**Comment by ljharb** [comment]
I think that #13 needs fixing, and that definitely an enzyme matcher should throw if anything but an enzyme wrapper is passed as the "actual".

**Comment by marcodejongh** [comment]
Yes I agree and that should be the fix for #11
